src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -1560,6 +1560,23 @@ services:
     tags:
       - { name: form.type }
 
+  form.type.sell.product.pricing.specific_priority_type:
+    class: 'PrestaShopBundle\Form\Admin\Sell\Product\Pricing\SpecificPricePriorityType'
```

Comment:
I think its nice to wrap these 2 fields in sepearate type, as they belong together. Lets say  someone wants to remove priorities form from product, they still have to remove both fields, because separately those fields are not usable. So I go with renaming option :smile: